charts/kubewarden-defaults/values.yaml

```diff
@@ -207,7 +207,7 @@ recommendedPolicies:
   userGroupPolicy:
     module:
       repository: "kubewarden/policies/user-group-psp"
-      tag: v0.6.0
+      tag: v0.6.1
```

Comment:
Let's wait a little bit. We have almost release another version. https://github.com/kubewarden/user-group-psp-policy/pull/97

Reply:
https://github.com/kubewarden/user-group-psp-policy/releases/tag/v0.6.2